### ISSUE TITLE
fx.prototype.cur will cover jQuery original fx.prototype.cur method

### ISSUE THREAD

**Issue by aaron-baby** [stevenwanderski/bxslider-3#63]
it's should be comment, may be cause slideDown error
in jQuery version 1.7, 1.7.1
//  jQuery.fx.prototype.cur = function(){
//      if ( this.elem[this.prop] != null && (!this.elem.style || this.elem.style[this.prop] == null) ) {
//          return this.elem[ this.prop ];
//      }
//
//      var r = parseFloat( jQuery.css( this.elem, this.prop ) );
        // return r && r > -10000 ? r : 0;
//      return r;
//  }

**Comment by jCrip** [comment]
Thanks!!! that fix a issue that i have with jquery UI hide property

**Comment by cmwhiting** [comment]
Yup, that fixed mine too! I'm not even sure why it tries to redefine the fx prototype, but commenting it out worked great.
